schema.sql

```diff
@@ -330,6 +330,15 @@ CREATE TABLE IF NOT EXISTS `tile_store` (
   FOREIGN KEY (`house_id`) REFERENCES `houses` (`id`) ON DELETE CASCADE
 ) ENGINE=InnoDB;
 
+CREATE TABLE IF NOT EXISTS `live_casts` (
```

Comment:
> although it would also need a column to check if said player is in fact live casting.

The columns `spectator_count` or `cast_password` could have a null value to represent "cast not active". Etc:
```sql
CREATE TABLE IF NOT EXISTS `players_online` (
  `player_id` int(11) NOT NULL,
  `cast_password` char(16) DEFAULT NULL,
  `cast_spectators` smallint(5) DEFAULT 0,
  PRIMARY KEY (`player_id`)
) ENGINE=MEMORY;
```

null = no cast
empty string = cast without password
string = cast with password

Sample that returns player name that are currently casting and their spectator count:
```sql
SELECT `p`.`name`, `o`.`cast_spectators`
FROM `players_online` AS `o`
INNER JOIN `players` AS `p`
ON `o`.`player_id` = `p`.`id`
WHERE `o`.`cast_password` != 'NULL';
```